resources/multisite_drupal_standard.make

```diff
@@ -1167,12 +1163,13 @@ projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
 projects[ec_resp][download][tag] = 2.3.9
 
 projects[atomium][type] = theme
-projects[atomium][version] = 2.12
+projects[atomium][version] = 2.22
```

Comment:
The last atomium release is 2.21: https://www.drupal.org/project/atomium/releases

Reply:
We are going to release Atomium 2.22 tomorrow, I was taking some advance as Jenkins is still down ;-)

Reply:
Atomium 7.x-2.22 has been released, closing this.